mucommander-translator/src/main/resources/dictionary.properties

```diff
@@ -484,6 +484,8 @@ search_dialog.case_sensitive = $[file_selection_dialog.case_sensitive]
 search_dialog.matches_regexp = $[file_selection_dialog.matches_regexp]
 search_dialog.wildcards = (* = any string, ? = any character, \\ = escape for literals: *?)
 search_dialog.search_text  = Containing text
+search_dialog.search_memonic=t
```

Comment:
this line should be placed elsewhere - here we should have a translatable strings whereas this looks like some kind of a configuration. we have configuration for keyboard shortcuts but they are associated with actions, I believe that this kind of shortcuts is not configurable anywhere - making it configurable should probably be out of the scope of this PR

Reply:
I have removed it from the dictionary and set in in the code.